services/core-api/app/flask_jwt_oidc_local/jwt_manager.py

```diff
@@ -166,6 +166,12 @@ def contains_role(self, roles):
             return True
         return False
 
+    def get_user_name(self):
```

Comment:
What would happen here if this was run for an idir user?
If it's just intended for bceid I'm fine with renaming, otherwise I'd expect this function to work for either, especially because it's a bit obscured by username() in access_decorators.

Reply:
This is only intended to be used for bceid since it just references the `bceid_username` within the token.  I think renaming it makes sense.

access_decorators was the most appropriate place I could find for this to live since it's pulling info from the JWT, but with the rename it should be clearer.